.github/workflows/pkg-pr-new.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish Any Commit
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: mghayes/multi-postgres:v2
+        ports:
+          - '5455:5432'
+        env:
+          POSTGRES_USER: prisma
+          POSTGRES_PASSWORD: prisma
+          POSTGRES_MULTIPLE_DATABASES: tests,drizzle
+
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: pnpm/action-setup@v4
+        name: Install pnpm
+        id: pnpm-install
+        with:
+          run_install: false
+
+      - name: Use Node.js 22
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          cache: 'pnpm'
+          cache-dependency-path: pnpm-lock.yaml
+
+      - name: Setup turbo cache
+        uses: actions/cache@v3
+        with:
+          path: node_modules/.cache
+          key: turbo-${{ runner.os }}-18-${{ hashFiles('**/pnpm-lock.yaml') }}
```

Comment:
I think this 18 used to refer to the node version, which is now out of sync.  

Should this share a turbo cache with other jobs? You might want a different key here